variables.tf

```diff
@@ -339,6 +339,11 @@ variable "etcd_lb_access" {
   default     = "private"
 }
 
+variable "etcd_assign_private_ip" {
```

Comment:
Also, I think `etcd_assign_private_ip` is misleading since _all_ instances are assigned a private IPs. A more informative name might be something like `maintain_etcd_private_ips`.